ansible/roles/openshift_sso_app_zabbix/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "item: openshift.master.cluster.synthetic.exists"
+  zbx_host:
+    zbx_server: "{{ ossoz_zbx_server }}"
+    zbx_user: "{{ ossoz_zbx_user }}"
+    zbx_password: "{{ ossoz_zbx_password }}"
+    state: list
+    hostname: "{{ oss_clusterid }}-synthetic"
+  register: out
+
+- name: "item: oso.sso.app.aggregate"
+  when: "{{ out.results.results | length == 1 }}"
+  zbx_item:
+    zbx_server: "{{ ossoz_zbx_server }}"
+    zbx_user: "{{ ossoz_zbx_user }}"
+    zbx_password: "{{ ossoz_zbx_password }}"
+    name: oso.sso.app.aggregate
+    key: 'grpsum[ops-aws,sso.monitoring.container.running,sum,15m]'
+    value_type: int
+    data_type: decimal
+    description: "The sum of oso-monitor-sso containers checking in."
+    multiplier: ''
+    hostname: "{{ item }}"
+    zabbix_type: aggregate
+    interval: 900
+    delta: 0
+    units: '%'
+  with_items: "{{ ossoz_masters }}"
+  delegate_to: "{{ oss_clusterid }}-synthetic"
```

Comment:
See above comment. Definitely don't delegate_to the synthetic host since it doesn't exist and ansible won't be able to ssh to it to run this task.